src/index.js

```diff
@@ -1,4 +1,6 @@
 import './style.css';
+import Scores from './modules/object.js';
+import { upload, get } from './modules/API.js';
 
 /* eslint-disable no-undef */
```

Comment:
Please try not to disable linters. Here you might have no option but if you really need to disable linter for a specific lines, cover with those line with another ``/* eslint-disable no-undef */`` so the rest of the file will be isolated from disabling.